docs/source/CppUsage.md

```diff
@@ -99,11 +99,19 @@ construction, access and mutation.
 To use:
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~{.cpp}
-    auto monsterobj = GetMonster(buffer)->UnPack();
-    cout << monsterobj->name;  // This is now a std::string!
-    monsterobj->name = "Bob";  // Change the name.
+    MonsterT monsterobj{GetMonster(buffer)};
```

Comment:
Note that we compile on older compiler such as GCC 4.6.3 and VS2010, so this syntax won't fly.